lib/libvarnish/vsa.c

```diff
@@ -174,6 +176,25 @@ struct suckaddr {
 
 const int vsa_suckaddr_len = sizeof(struct suckaddr);
 
+/*
+ * Bogus IPv4 address 0.0.0.0:0 to be used for VCL *.ip variables when the
+ * "real" address is not IP (such as UDS addresses).
+ */
+static struct suckaddr bogo_ip_vsa;
+const struct suckaddr *bogo_ip = &bogo_ip_vsa;
+
+void
+VSA_Init()
+{
+	struct addrinfo hints, *res = NULL;
+
+	memset(&hints, 0, sizeof(hints));
+	hints.ai_family = AF_INET;
```

Comment:
Add `AI_NUMERICHOST` and `AI_NUMERICSERV` flags?

Reply:
Yes, thought about that after the commit, we should have it.